task on arrays/script.js

```diff
@@ -0,0 +1,104 @@
+function isArray(obj) {
+  return typeof obj ==="object" &&
```

Comment:
imo `typeof obj ==="object"` is not required
in additional code styles is bad. for your case you could use
```
return (typeof obj ==="object" &&
  Object.prototype.toString.call(obj) === "[object Array]");
```